crates/uv/tests/it/pip_compile.rs

```diff
@@ -13949,7 +13949,7 @@ fn invalid_platform() -> Result<()> {
 
           hint: Wheels are available for `open3d` (v0.15.2) with the following ABI tags: `cp36m`, `cp37m`, `cp38`, `cp39`
 
-          hint: Wheels are available for `open3d` (v0.18.0) on the following platforms: `macosx_11_0_x86_64`, `macosx_13_0_arm64`, `manylinux_2_27_aarch64`, `manylinux_2_27_x86_64`, `win_amd64`
+          hint: Wheels are available for `open3d` (v0.18.0) on the following platforms: `macosx_11_0_x86_64`, `macosx_13_0_arm64`, `manylinux_2_27_aarch64`, `manylinux_2_27_x86_64`, `win_amd64`. The closest match is `cp310-cp310-win_amd64`.
```

Comment:
I'm not sure these help.

In this example, I get confused by the additional tags (e.g., `cp310`). Maybe it'd be helpful as "The closest match to your platform (e.g., ...) is ...".

Reply:
Separately, how is `cp310-cp310-win_amd64` the closest match to `manylinux_2_17_x86_64`?

Reply:
It should probably be `cp310-cp310-manylinux_2_27_x86_64 `. You don't find that helpful?

Reply:
I don't think so. Maybe if I _also_ had a Python compatibility error? I'm just not sure what I could do with that information, and a quick scan of the available tags is already showing where I've gone wrong.

What does knowing the closest tag change? Does it make a particular action clearer?